cpp/tests/replace/clamp_test.cpp

```diff
@@ -262,7 +262,7 @@ TYPED_TEST(ClampFloatTest, WithNANandNoNull) {
     using T = TypeParam;
     using ScalarType = cudf::experimental::scalar_type_t<T>;
 
-    cudf::test::fixed_width_column_wrapper<T> input({8.0, 6.0, NAN, 3.0, 4.0, 5.0, 1.0, NAN, 2.0, 9.0});
+    cudf::test::fixed_width_column_wrapper<T> input({T(8.0), T(6.0), T(NAN), T(3.0), T(4.0), T(5.0), T(1.0), T(NAN), T(2.0), T(9.0)});
```

Comment:
Too verbose `T()` for every value. Any less verbose alternative?

Reply:
It is a lot less verbose than `static_cast<T>()`